src/pkg/repository/repository.go

```diff
@@ -229,23 +230,21 @@ func Connect(
 		return nil, clues.Wrap(err, "constructing event bus")
 	}
 
-	rm := &repositoryModel{}
-
 	// Do not query repo ID if metrics are disabled
 	if !opts.DisableMetrics {
-		rm, err = getRepoModel(ctx, ms)
-		if err != nil {
-			return nil, clues.New("retrieving repo info")
+		repoID := "not_found"
```

Comment:
We shouldn't initialize to `not_found` here. If there aren't any code paths where this is retrieved in this function block - we don't need that

Reply:
I would look into where we initialize `cfg` and initialize to a constant before we retrieve it from the config file